share/management/commands/disambiguate.py

```diff
@@ -1,30 +1,22 @@
-from django.apps import apps
 from django.core.management.base import BaseCommand
-from django.conf import settings
 
-from share.models import ShareUser, NormalizedData
-from share.tasks import DisambiguatorTask
+from share.models import NormalizedData
+from share.tasks import disambiguate
 
 
 class Command(BaseCommand):
 
     def add_arguments(self, parser):
-        parser.add_argument('source', type=str, help='The name of the provider to run')
         parser.add_argument('normalized', nargs='*', type=int, help='The id(*) of the normalized data to make changes for')
-        parser.add_argument('--all', action='store_true', help='Make changes for all normalized data for the source specified')
+        parser.add_argument('--source', type=str, help='Name of a Source, to disambiguate ALL its normalized data')
```

Comment:
Ohhhh for some reason I read it with a comma after ALL. Which I was thinking meant `manage.py disambiguate --source="normalized data"`. For my sake, could you rephrase that help text to something else, e.g. 'Name of a Source, to disambiguate ALL normalized data for that source'